Odata-docs/webapi-8/tutorials/custom-skiptokenhandler.md

```diff
@@ -0,0 +1,549 @@
+---
+title:  "Implementing a Custom Skip Token Handler"
+description: A tutorial on how to customize generated skip token with a custom SkipTokenHandler class.
+date:   2023-01-19
+ms.date: 01/19/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Implementing a Custom Skip Token Handler
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+Skip tokens are used in server-side paging to keep track of the last record that was sent the client so that it can generate the next page of results. The skip token is opaque to the client, this means that the server has freedom to decide what the contents of the skip token should be and how to interpret them. ASP.NET Core OData generates by default the skip token by combining the fields and values of the the a reference record. However, you can override this behavior by providing your own custom logic for generating and processing skip tokens.
```

Comment:
```suggestion
Skip tokens are used in server-side paging to keep track of the last record that was sent the client so that it can generate the next page of results. The skip token is opaque to the client, this means that the server has freedom to decide what the contents of the skip token should be and how to interpret them. ASP.NET Core OData generates by default the skip token by combining the fields and values of a reference record. However, you can override this behavior by providing your own custom logic for generating and processing skip tokens.
```